### DISCUSSION TITLE
shader window location

### DISCUSSION THREAD

**Discussion by Cristian260121** [bonsai-rx/bonsai#1331] [Q&A]
Hi I am starting to use bonvision. 
When I start the stimuli, the shader window location begins in the middle of the screen. How can I do that shader window start on the left side of the screen?

**Comment by bruno-f-cruz** [comment]
You can change the property named `Location` in the `CreateWindow` operator.

**Reply by Cristian260121** [reply]
Dear:
I have tried to change the value of the location, but I need to know what kind of number format to put in the "location properties". Do you know the number format?

**Comment by bruno-f-cruz** [comment]
Format should be int,int (pixels) in relation to the top left of the screen

**Reply by Cristian260121** [reply]
Dear

When I put 
300 , 200 
it does not work, the bonsai indicate that 

300,300 no es un valor válido para Int32

do you know what it could mean?

**Reply by bruno-f-cruz** [reply]
I guess you are running a Spanish version of windows. Can you try to change the decimal separator to not use commas? That should fix it.

**Reply by Cristian260121** [reply]
thanks for your help 
it was very useful
